.github/.dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
```

Comment:
Dependabot can't update packages.config, so this needs to be excluded in the .reposync.yml instead.

EDIT: Actually, repo sync is disabled, so this is just manually added I guess. Just need to remove it from the PR then.

Reply:
Good point, @bording. After more thought, I'm going to see if we can skip this repo altogether, since it's deprecated.